app/models/account.rb

```diff
@@ -3,7 +3,7 @@
 #
 # Table name: accounts
 #
-#  id                      :bigint           not null, primary key
+#  id                      :integer          not null, primary key
 #  username                :string           default(""), not null
```

Comment:
Huh? Why would we go back to integer?

Reply:
These model annotations are auto-generated. I've deleted the rest of your comments because they're not a concern.

Reply:
Right. Still, noisy for no reason, and I don't understand why they would be generated that way? `db/schema.rb` seems fine, though.

Reply:
see the conversation on #5461

Reply:
Every db:migrate is gonna reset these to this. So I'd rather have them wrong than deal with noise in every PR. I shouldn't have merged #5461 but I did not realize annotate would not pick up the correct types, thought something just got forgotten because of the unusual way we performed the snowflake migrations.